lib/package.json

```diff
@@ -1,23 +1,35 @@
 {
   "name": "lc-form-validation",
-  "version": "0.0.1",
-  "description": "lcFormValidation is form library validation: heavily based on javascript (no html attributes / annotions), async by default. lcFormValidation is third party / framework agnostic, it can be easily integrated with frameworks likeReact.",
+  "version": "0.1.1",
+  "description": "lcFormValidation is form library validation: heavily based on javascript (no html attributes / annotations), async by default. lcFormValidation is third party / framework agnostic, it can be easily integrated with frameworks like React.",
   "main": "lc-form-validation.js",
```

Comment:
this main file is where webpack going to search when we import a node_module. What if we are in production environment? Does it require this file instead of minified?